### ISSUE TITLE
Tapping on unclosed public_transport=platform ways is not functional

### ISSUE THREAD

**Issue by jengelh** [osmandapp/Osmand#8498]
>Please give us the following information so that we can try to **reproduce** your issue:

Observed:

Go to location 51.54619 9.88602 to find way 327681321. This is an _area_ (closed loop). An icon is shown. A text label is shown. In OsmAnd, the label/icon can be tapped to reveal more public transport information such as routes.

Go to location 51.54658 9.88632 to find way 319930864. This is a _way_/line. No icon is shown (the icon nearby is from compatibility node 3264239265). A label is shown for the platform. The label cannot be tapped; nothing happens.

Unclosed platforms are legit however, according to https://wiki.openstreetmap.org/wiki/Tag:public_transport%3Dplatform .

Expected to see instead:

Allow tapping on non-area platforms.

>What version of OsmAnd are you using, on what device and Android/iOS version?

Android 9, OsmAnd+ 3.5.5 (2019-11-28)

>Do you use the offline maps offered within the OsmAnd app for download, or online (tile / raster) maps?

Monthly-updated OBF download from within the app.

>If you have an issue related to offline maps, tell us the exact name of the map file where the issue occurs, and its edition date.

"Germany Lower Saxony" 2020-Feb-01; the mapdata is fine though.

**Comment by vshcherb** [comment]
Yes we don't display icon on the ways that's why it is not possible to tap on it. Though it should be possible to tap on public_transport=stop and see the transport which stops there.

**Comment by jengelh** [comment]
No icon is rendered for public_transport=stop_position, so it cannot be tapped either. (see node 3264237165 for example)

**Comment by vshcherb** [comment]
As I see people still add bus_stop to get it working here https://www.openstreetmap.org/node/3264239265. Probably this is an easy solution for this problem

**Comment by jengelh** [comment]
Yeah I'd like to get rid of the extra bus_stop because that's just redundant information.

**Comment by vshcherb** [comment]
It is really strange situation, our engine doesn't support icons on the line for a good reason, it was needed for many years. 
As I see public transport = plaftorm area works good and we have icon in the middle though platform as the line represents typically a thin area in real world. 

We have shield tags that draw icons on the line (for example ref) and probably we can use that approach though it looks pretty interesting case.
